modules/account-lib/src/coin/stacks/utils.ts

```diff
@@ -0,0 +1,142 @@
+import {
+  AddressHashMode,
+  StacksTransaction,
+  TransactionVersion,
+  addressFromVersionHash,
+  addressHashModeToVersion,
+  addressToString,
+  validateStacksAddress,
+} from '@stacks/transactions';
+import BigNumber from 'bignumber.js';
+
+/**
+ * Encodes a buffer as a `0x` prefixed lower-case hex string.
+ *
+ * @param {Buffer} buff - a buffer with a hexadecimal string
+ * @returns {string} - the hexadecimal string prefixed with "0x"
+ */
+export function bufferToHexPrefixString(buff: Buffer): string {
+  return '0x' + buff.toString('hex');
+}
+
+/**
+ * @param hex
+ */
+export function removeHexPrefix(hex: string): string {
```

Comment:
merits in using `stripHexPrefix` from `ethereumjs-util`? Trying to avoid reinventing the wheel